src/Workspaces/Remote/ServiceHub/Host/AssetProvider.cs

```diff
@@ -22,7 +23,7 @@ namespace Microsoft.CodeAnalysis.Remote;
 internal sealed partial class AssetProvider(Checksum solutionChecksum, SolutionAssetCache assetCache, IAssetSource assetSource, ISerializerService serializerService)
     : AbstractAssetProvider
 {
-    private const int PooledChecksumArraySize = 256;
+    private const int PooledChecksumArraySize = 1024;
```

Comment:
i upped this as i was gonig over 256 a fair bit (easy when we have 900 projects in roslyn alone).